daemon/logs.go

```diff
@@ -30,15 +31,15 @@ func (daemon *Daemon) ContainerLogs(ctx context.Context, containerName string, c
 	})
 
 	if !(config.ShowStdout || config.ShowStderr) {
-		return nil, false, validationError{errors.New("You must choose at least one stream")}
+		return nil, false, errdefs.InvalidParameter(errors.New("You must choose at least one stream"))
 	}
 	container, err := daemon.GetContainer(containerName)
 	if err != nil {
 		return nil, false, err
 	}
 
 	if container.RemovalInProgress || container.Dead {
-		return nil, false, stateConflictError{errors.New("can not get logs from container which is dead or marked for removal")}
+		return nil, false, errdefs.InvalidParameter(errors.New("can not get logs from container which is dead or marked for removal"))
```

Comment:
Should this be `Conflict()` (I believe others were coverted to Conflict)

Reply:
Yes should be conflict.